incubator/fluentd-cloudwatch/README.md

```diff
@@ -62,6 +62,7 @@ The following tables lists the configurable parameters of the Fluentd Cloudwatch
 | `rbac.create`                   | If true, create & use RBAC resources       | `false`                                                    |
 | `rbac.serviceAccountName`       | existing ServiceAccount to use (ignored if rbac.create=true) | `default`                                |
 | `tolerations`                   | Add tolerations                            | `[]`                                                       |
+| `envs`                          | Add pod environment variables (must be specified as a single line object)             | `[]`
```

Comment:
Please rename this to `extraVars`. There are already quite some charts to use this. Let's be consistent.